lib/internal/errors.js

```diff
@@ -261,7 +261,7 @@ E('ERR_NO_LONGER_SUPPORTED', '%s is no longer supported');
 E('ERR_OUT_OF_RANGE', 'The "%s" argument is out of range');
 E('ERR_OUTOFMEMORY', 'Out of memory');
 E('ERR_PARSE_HISTORY_DATA', 'Could not parse history data in %s');
-E('ERR_QUERYSTRING_URI_INVALID', '%s');
+E('ERR_INVALID_URL', 'URI malformed');
```

Comment:
So we have a new lint rule that makes sure the error code are sorted in ASCIIbetical order.
It would have made you notice https://github.com/nodejs/node/blob/4518d94e6104323b06781ba608e59208d35a6129/lib/internal/errors.js#L243
So IMHO make this `ERR_INVALID_URI` (so that we can keep the message text the same), and move it to L243 above the previous `ERR_INVALID_URL`